src/services/refact/docker.ts

```diff
@@ -0,0 +1,301 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { RootState } from "../../app/store";
+import { DOCKER_CONTAINER_ACTION, DOCKER_CONTAINER_LIST } from "./consts";
+
```

Comment:
There might be some cache issues here,

Reply:
Added TODO to test it and adjust in the future